js/piwik.js

```diff
@@ -1464,22 +1465,26 @@ if (typeof window.Piwik !== 'object') {
 
         var content = {
             CONTENT_ATTR: 'data-track-content',
-            CONTENT_CLASS: 'piwikTrackContent',
+            CONTENT_CLASS: 'matomoTrackContent',
```

Comment:
will need to later update docs to mention these new attributes... I don't think we actually deprecate the old ones